prow/scripts/cluster-integration/kyma-gke-integration.sh

```diff
@@ -97,13 +97,31 @@ cleanup() {
     exit "${EXIT_STATUS}"
 }
 
+# Enforce lowercase
+readonly REPO_OWNER=$(echo "${REPO_OWNER}" | tr '[:upper:]' '[:lower:]')
+export REPO_OWNER
+readonly REPO_NAME=$(echo "${REPO_NAME}" | tr '[:upper:]' '[:lower:]')
+export REPO_NAME
+
+if [[ "$BUILD_TYPE" == "pr" ]]; then
+    # In case of PR, operate on PR number
+    IP_ADDRESS_NAME=$(echo "pr-${PULL_NUMBER}-job-${PROW_JOB_ID}" | tr "[:upper:]" "[:lower:]")
+    CLUSTER_NAME="gkeint-${REPO_OWNER}-${REPO_NAME}-${PULL_NUMBER}"
+    KYMA_INSTALLER_IMAGE="${DOCKER_PUSH_REPOSITORY}${DOCKER_PUSH_DIRECTORY}/gke-integration/${REPO_OWNER}/${REPO_NAME}:PR-${PULL_NUMBER}"
+else 
+    # Otherwise (master, or release), operate on triggering commit id
+    readonly COMMIT_ID=$(cd "$KYMA_SOURCES_DIR" && git rev-parse --short HEAD)
```

Comment:
could you check if we can replace COMMIT_ID with PULL_BASE_SHA provided by Prow?

Reply:
We could, however the env looks like `PULL_BASE_SHA=b25b58b147fab0c89413ca238858e17b7916e201`, so we would have to trim it